web/src/testgrid-dashboard-summary.ts

```diff
@@ -88,6 +90,7 @@ export class TestgridDashboardSummary extends LitElement {
       var tabSummaries: Array<TabSummaryInfo> = [];
       data.tabSummaries.forEach(ts => {
         const si = convertResponse(ts);
+        si.dashboardName = this.dashboardName
```

Comment:
I believe this can be done within `convertResponse` function

Reply:
Done.